app/views/feedback/thanks.html.erb

```diff
@@ -0,0 +1,14 @@
+<%- @page_title = 'Feedback' -%>
+
+<%= render partial: 'shared/contextual_header', locals: { page_title: @page_title, parent_pages: [] } %>
+
+<div class="usa-section main-section">
+  <h1><%= @page_title %></h1>
+
+  <div class="feedback-thanks-container">
+    <%= image_tag 'icons/feedback.svg', alt: '', role: 'presentation' %>
+
+    <p><b>Thanks for your feedback!</b></p>
+    <p>Your feedback has been sent to USTRANSCOM so that we can continue to improve the military relocation experience.</p>
```

Comment:
Bike-shedding some here, but what do you think about rewording this as:

> Your feedback has been sent to USTRANSCOM. Thank you for your input as we continue to improve the military relocation experience.

Reply:
Sounds good to me!